api/metrics-api-jaxrs-1.1/src/main/webapp/WEB-INF/web.xml

```diff
@@ -30,17 +30,19 @@
   </context-param>
 
   <context-param>
-    <param-name>resteasy.media.type.mappings</param-name>
```

Comment:
Could we remove this from the other web.xml as well so that we keep differences in the 2 impl to a minimum?